### PR TITLE
Gltf boombox example

### PR DESCRIPTION
**Pull request by kwokcb**
- Add in BoomBox example from Ed. Minor name change to shader and material so it's easier to read results
- Add in `GltfPbr` directory for generation tests and render tests.

## Default test result. GLSL and OSL
image

## Boombox test result. GLSL and OSL
image

## Boombox viewer result. GLSL
image

### Test Run #
[glTF_test_run_Mar_15_2022.pdf](https://github.com/kwokcb/MaterialX/files/8256285/glTF_test_run_Mar_15_2022.pdf)
#

### CODE REVIEW

File: resources/Materials/Examples/GltfPbr/gltf_pbr_boombox.mtlx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <output name="out_emission" type="color3" nodename="image_emission" />
+    <output name="out_normal" type="vector3" nodename="normalmap" />
+    <output name="out_occlusion" type="float" nodename="extract_occlusion" />
+    <output name="out_roughness" type="float" nodename="extract_roughness" />
+    <output name="out_metallic" type="float" nodename="extract_metallic" />
+  </nodegraph>
+  <gltf_pbr name="SR_boombox" type="surfaceshader">
+    <input name="base_color" type="color3" nodegraph="NG_boombox" output="out_basecolor" />
+    <input name="metallic" type="float" nodegraph="NG_boombox" output="out_metallic" />
+    <input name="roughness" type="float" nodegraph="NG_boombox" output="out_roughness" />
```
Comment (emackey) [inline resources/Materials/Examples/GltfPbr/gltf_pbr_boombox.mtlx:47]: Now that we have a stub for `occlusion`, we should hook it up here.

```suggestion
    <input name="roughness" type="float" nodegraph="NG_boombox" output="out_roughness" />
    <input name="occlusion" type="float" nodegraph="NG_boombox" output="out_occlusion" />
```

Comment (emackey) [reply resources/Materials/Examples/GltfPbr/gltf_pbr_boombox.mtlx:47]: /cc https://github.com/KhronosGroup/MaterialX/pull/1

Comment (kwokcb) [reply resources/Materials/Examples/GltfPbr/gltf_pbr_boombox.mtlx:47]: > Now that we have a stub for `occlusion`, we should hook it up here.

Added update.

Comment (emackey) [reply resources/Materials/Examples/GltfPbr/gltf_pbr_boombox.mtlx:47]: Apparently I cannot mark this as "resolved," but looks good to me.

**Comment by kwokcb** [comment]
I've moved this from this staging PR to a merge to ASWF PR (https://github.com/AcademySoftwareFoundation/MaterialX/pull/870).

**Pull request closed without merging**